src/services/vendors/GPIB.ts

```diff
@@ -23,7 +23,7 @@ export class GPIBVendor implements IVendor {
             lastName,
             email,
             password,
-            referralCode: "",
+            referralCode: this.config.get(ConfigSettings.GPIB_REFERRAL_CODE),
```

Comment:
i would argue that we put this in a const called `referralCode` and also check that one has been provided, otherwise throw an error - i assume that we don't want any signups without getting our cut

Reply:
good point